2_process/src/filter_sites_custom.R

```diff
@@ -2,22 +2,23 @@
 #' Subset the site list to ones relevant for the storm
 #'
 #' @param ind_file character file name where the output should be saved
-#' @param sites_file path for data.frame of sites with automatic filtering applied already
-filter_sites_custom <- function(ind_file, sites_file) {
+#' @param sites_ind indicator file for an sf data.frame of sites with automatic filtering applied already
+filter_sites_custom <- function(ind_file, sites_ind) {
 
-  sites <- readRDS(scipiper::sc_retrieve(sites_file))
+  # get sites data frame with NWS data
+  sites_df <- readRDS(sc_retrieve(sites_ind))
 
   # subset the sites from the wide net cast to ones relevant to the storm
-  # subset criteria TBD
-  sites_info <- dataRetrieval::readNWISsite(siteNumbers = sites)
+  # subset criteria TBD for each storm
+  sites_info <- dataRetrieval::readNWISsite(siteNumbers = sites_df$site_no)
 
   sites_info_subset <- sites_info %>%
     # only include sites with large drainage areas
     filter(drain_area_va > quantile(drain_area_va, 0.75, na.rm=TRUE))
```

Comment:
this is fine for this demo custom function, but we should keep a close eye on `drain_area_va` when we get to real storms, because i don't think NWIS reliably supplies drain_area_va for every site.